src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -55,12 +68,48 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
             body = needsParentheses(actualExpression) ? createParen(actualExpression) : actualExpression;
             suppressLeadingAndTrailingTrivia(body);
             copyLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            copyTrailingAsLeadingComments(returnStatement, body, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);
+            // Copy the trailing comments after the return statement
+            // Add them to our trailingCommentsHolder to use later
+            copyTrailingComments(returnStatement, trailingCommentsHolder, file, SyntaxKind.MultiLineCommentTrivia, /* hasTrailingNewLine */ false);

```

Comment:
We need the trailing comment after the return statement. So we use this function that someone else wrote to do it. 

Example (in case it helps):
```javascript
const f = () => { return a; /* don't leave me behind */}
```